app/src/main/java/com/abisoft/todocompose/TodoItemsRepository.kt

```diff
@@ -1,79 +1,65 @@
 package com.abisoft.todocompose
 
+import com.abisoft.todocompose.api.UpdateRequestData
+import com.abisoft.todocompose.model.TodoItemNetwork
+import com.abisoft.todocompose.model.TodoItemPost
 import com.example.myapplication.okhttp_client.TodoApiService
 import com.example.myapplication.okhttp_client.TodoDto
+import retrofit2.Response
 
+@Suppress("UNREACHABLE_CODE")
 class TodoItemsRepository(private val apiService: TodoApiService) {
```

Comment:
nit: good practice to wrap network logic to separate DataSource class. In this case repository only operates merging and organizing data flow logic